src/test/unit/views/manual-upload.test.ts

```diff
@@ -98,9 +99,10 @@ describe('Manual upload page', () => {
     });
 
     it('should display display date question', () => {
-      const displayDate = formElements.getElementsByClassName(fieldSetClass)[1];
-      expect(displayDate.innerHTML).contains(expectedDisplayDates, 'Could not find inset content date question');
-      expect(displayDate.getElementsByClassName(dateInputClass).length).equals(2, 'Could not find inset content date');
+      const displayDateFrom = formElements.getElementsByClassName(fieldSetClass)[1];
+      const displayDateTo = formElements.getElementsByClassName(fieldSetClass)[2];
+      expect(displayDateFrom.innerHTML).contains(expectedDisplayDateFrom, 'Could not find inset content date question');
```

Comment:
I believe the messages on line 104 / 105 should be updated from content date to display from / display to